src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,102 @@
+package duke;
+
+/**
+ * Class that is a chat bot that can store tasks.
+ */
+public class Duke {
+    private boolean isRunning = true;
+    private enum Commands { list, done, todo, event, deadline, delete, find }
```

Comment:
Commands seem to be used in multiple files (Duke.java and Parser.java). May I suggest abstracting it to a Commands.java enum file? If not you may end up updating one without doing so for the other.